.github/workflows/generate_reference_test_data.yml

```diff
@@ -36,21 +36,23 @@ jobs:
         export PATH=$(pwd)/openmpi-4.1.4/installed/bin/:$PATH
         echo "mpif90 version: $(mpif90 --version | head -1)" >> metadata.txt
         echo "mpirun version: $(mpirun --version | head -1)" >> metadata.txt
-        mv metadata.txt test/data/Taylor-Green-Vortex/
```

Comment:
Does the `data` subdirectory not help keep this tidy?

Reply:
Since i moved all the tests under CTest in the tests folder, i took the decision to have all files related to a test in the specific directory. Let me know if this make sense